src/components/DatePicker.vue

```diff
@@ -130,6 +132,10 @@
     components: { Day },
 
     props: {
+      borderDateColor:{
```

Comment:
maybe this prop should also be called `currentDateStyle`? Then we could just pass it to the day componet with `:currentDateStyle="currentDateStyle"`